File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/postgresql/NumericTypes.scala
package dev.typr.foundationssc.docs.postgresql
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

@SuppressWarnings(Array("unused"))
object NumericTypes:
  //start
  val intType: PgType[Int] = PgTypes.int4
  val decimalType: PgType[BigDecimal] = PgTypes.numeric
  val moneyType: PgType[Money] = PgTypes.money
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/postgresql/NumericTypes.kt
package dev.typr.foundationskt.docs.postgresql

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*
import java.math.BigDecimal

@Suppress("unused")
class NumericTypes {
    //start
    val intType: PgType<Int> = PgTypes.int4
    val decimalType: PgType<BigDecimal> = PgTypes.numeric
    val moneyType: PgType<Money> = PgTypes.money
    //stop
}
